winpty/src/windows.rs

```diff
@@ -127,6 +143,21 @@ impl Drop for Config {
 /// A struct representing the winpty agent process
 pub struct Winpty(*mut winpty_t);
 
+pub struct ChildHandles {
+    process: HANDLE,
+    thread: HANDLE
+}
+
+impl ChildHandles {
+    pub fn process(&self) -> HANDLE {
+        self.process
+    }
+
+    pub fn thread(&self) -> HANDLE {
+        self.thread
+    }
+}
```

Comment:
I don't think there's much reason to have methods for accessing `process` and `thread` here, right? We can just make the fields `pub` and that shouldn't be a problem.

Reply:
Ok 👍